packages/common/src/services/audius-backend/AudiusBackend.ts

```diff
@@ -717,8 +717,12 @@ export const audiusBackend = ({
   }
 
   function getEthWeb3Config() {
-    const providerUrls =
-      getRemoteVar(StringKeys.ETH_PROVIDER_URLS) || ethProviderUrls
+    // In a dev env, always ignore the remote var which is inherited from staging
+    const isDevelopment = process.env.REACT_APP_ENVIRONMENT === 'development'
```

Comment:
So with Audius-backend we are trying to make it env agnostic. The pattern is to put this in the env.ts file in common, and pass env into Audius backend. Lmk if I can help here

Reply:
ah good call. thx @dylanjeffers . i was not observant enough here